tests/resources/Harbor-Pages/Project.robot

```diff
@@ -118,11 +111,15 @@ Make Project Public
 
 Delete Repo
     [Arguments]  ${projectname}
-    Click Element  xpath=//clr-dg-row[contains(.,"${projectname}")]//clr-checkbox-wrapper//label
-    Wait Until Element Is Enabled  //button[contains(.,"Delete")]
-    Click Element  xpath=//button[contains(.,"Delete")]
-    Wait Until Element Is Visible  //clr-modal//button[2]
-    Click Element  xpath=//clr-modal//button[2]
+    ${element_repo_checkbox}=  Set Variable  xpath=//clr-dg-row[contains(.,"${projectname}")]//clr-checkbox-wrapper//label
+    Retry Keyword With Element When Error  Wait Until Element Is Visible And Enabled  ${element_repo_checkbox}
+    Retry Keyword With Element When Error  Click Element  ${element_repo_checkbox}
+    ${element_delete_btn}=  Set Variable  xpath=//button[contains(.,"Delete")]
+    Retry Keyword With Element When Error  Wait Until Element Is Visible And Enabled  ${element_delete_btn}
+    Retry Keyword With Element When Error  Click Element  ${element_delete_btn}
+    ${element_delete_confirm_btn}=  Set Variable  xpath=//clr-modal//button[2]
```

Comment:
please move it to the project_elements.robot, but it's not a blocker for this PR.